Header header

int16 station_id
int16 station_health
float32 observation_latency

float32 ref_x
float32 ref_y
float32 ref_z

KeyInt[] rx_msgs
